### PR TITLE
remove dead projects

### PR DESCRIPTION
**Pull request by fiatjaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,13 +49,9 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [IPFS Drive](http://ipfs-drive.ydns.eu) - In browser file manager for IPFS. [GitHub](https://github.com/fazo96/ipfs-drive)
 * [ipfs.ink](https://ipfs.ink) - Publish and render markdown essays to and from ipfs. [GitHub](https://github.com/kpcyrd/ipfs.ink)
 * [ipfs-search](http://ipfs-search.com) - Search engine for files and directories on IPFS.
-* [ipfs-share](https://github.com/rameshvarun/ipfs-share) - Pastebin/Image host/File sharing application
-* [ipfs.pics](https://github.com/ipfspics/ipfspics-server) - Upload and share pics.
-* [IPFSBin](https://github.com/victorbjelkholm/ipfsbin) - Pastebin clone build.
```
Comment (Stebalien) [inline README.md]: @VictorBjelkholm

**Comment by pokeball99** [comment]
why should these "dead" projects be removed? the git repos still work just fine

**Comment by fiatjaf** [comment]
Have you clicked them all?

**Comment by fiatjaf** [comment]
Actually, this repository is more like a list of "everything done in IPFS" than actually "awesome" IPFS projects.

I think a lot of old and abandoned stuff should be removed (much more than I removed in this PR) if the repository is to serve the purposes it should be serving. This PR was a push on that direction. Of course it should be discussed and not merged as it is, but not dismissed. Something must be done, instead of just leaving the things as they are.

If you're going to classify as "awesome" a git repository that's just a half-finished idea of how to do something not very useful, then everything is awesome and we're again lost as if this repository was never started.

**Comment by koalalorenzo** [comment]
it seems that nobody is taking care of this repository! I think this is a great resource (should I say awesome?) for people! There are a lot of interesting PR that are stuck!

Can we/anybody help with this?

**Comment by Stebalien** [comment]
> I think a lot of old and abandoned stuff should be removed (much more than I removed in this PR) if the repository is to serve the purposes it should be serving. This PR was a push on that direction. Of course it should be discussed and not merged as it is, but not dismissed. Something must be done, instead of just leaving the things as they are.

I agree that something should be done. However, many of theses projects *do* serve as good examples of what one can do with IPFS (that's effectively the point of this repo). Just because the developer lost interest doesn't make the project any less useful.

Maybe add a "defunct" category?

**Comment by fiatjaf** [comment]
Looks like a good idea. Or somehow separate between "Interesting ideas" and "Actually working stuff".

**Comment by VictorBjelkholm** [comment]
It's worth discussing though! Please open a issue instead and we can hopefully reach a conclusion. Closing this in the meantime as we've changed the format of contributions made to awesome-ipfs and we're not sure if we should remove/label defunct projects.

**Pull request closed without merging**